.github/workflows/stage-bump-version.yml

```diff
@@ -0,0 +1,82 @@
+# Takes in how to bump version as input
+# Commits changes to bump version and outputs the new version and commit hash as output
+
+name: Bump version
+
+permissions:
+  # This is required for requesting the OIDC token
+  id-token: write
+  contents: read
+
+on:
+  workflow_call:
+    secrets:
+      desk_mood:
+        required: true
+      king_health:
+        required: true
+    outputs:
+      new_dev_version:
+        value: ${{ jobs.get-new-dev-version.outputs.new_dev_version }}
+      new_stage_version:
+        value: ${{ jobs.get-new-stage-version.outputs.new_stage_version }}
+      bump_sha:
+        value: ${{ jobs.update-version-in-repo.outputs.bump_sha }}
+
+jobs:
+  get-next-stage-jfrog-version:
+    name: Get the latest stage version from JFrog
+    runs-on: ubuntu-22.04
+    outputs:
+      stage_version: ${{ steps.get-next-stage-jfrog-version.outputs.stage_version }}
+    steps:
+    # Checkout the branch where we want to bump the new version
+    - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      with:
+        token: ${{ secrets.desk_mood }}
+
+    - name: Set up JFrog credentials
+      uses: jfrog/setup-jfrog-cli@88e9eba31c07e31beefa4cef5c0e93d1af9535d7 # v4.6.1
+      env:
+        JF_URL: ${{ secrets.king_health }}
+      with:
+        oidc-provider-name: gh-aerospike-clients
+        oidc-audience: aerospike/clients
+
+    - name: Get jfrog version
+      id: get-next-stage-jfrog-version
+      run: |
+        echo "stage_version=$(jfrog rt s "clients-npm-stage-local/aerospike/*" | jq -r '.[]?.path | capture("aerospike/(?<version>[^/]+)") | .version' | grep -v '^-$' | grep -v '^lib$' | sort -V | tail -n1)" >> $GITHUB_OUTPUT
+
+    - name: Print jfrog version
+      run: echo "Jfrog version is ${{ steps.get-next-stage-jfrog-version.outputs.stage_version }}"
+
+  get-new-stage-version:
+    runs-on: ubuntu-22.04
+    needs: [
+      get-next-stage-jfrog-version
+    ]
+    outputs:
+      new_stage_version: ${{ steps.get-new-version.outputs.new_stage_version }}
+    steps:
+
+    - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      with:
+        token: ${{ secrets.desk_mood }}
+
+    - name: Install library that parses nodejs versions
+      run: npm install semver --ignore-scripts
```

Comment:
Let's pin the version of `semver` we're installing.

Reply:
Pinned the version in next commit.